lib/virtfs/ext3/file_data.rb

```diff
@@ -68,17 +68,17 @@ def write(buf, _len = buf.length)
     private
 
     def getStartBlock(pos, len)
-      startBlock, startByte = pos.divmod(@blockSize)
-      endBlock, endByte = (pos + len - 1).divmod(@blockSize)
+      startBlock, startByte = pos.divmod(@block_size)
+      endBlock, endByte = (pos + len - 1).divmod(@block_size)
```

Comment:
If you're going through the incredibly tedious chore of aligning things and getting rid of camel-case when possible maybe the "=" here should line up with the previous line?